actors/evm/src/lib.rs

```diff
@@ -189,14 +192,15 @@ impl EvmContractActor {
     pub fn handle_filecoin_method<RT>(
         rt: &mut RT,
         method: u64,
-        codec: u64,
-        params: &[u8],
+        args: Option<IpldBlock>,
     ) -> Result<Vec<u8>, ActorError>
     where
         RT: Runtime,
         RT::Blockstore: Clone,
     {
-        let input = handle_filecoin_method_input(method, codec, params);
+        // TODO: Raw feels like the correct-est thing here? Is there an "I'm empty" codec?
+        let params = args.unwrap_or(IpldBlock { codec: IPLD_RAW, data: vec![] });
```

Comment:
"0", at the moment. Technically it's the identity hash function, but I've been using it for the empty block. I.e.:

- Codec: 0.
- BlockID: 0.

Reply:
Sure, that works -- I would like to define that somewhere, but it might be funky to do that if 0 is "actually" the identity hash function.

Reply:
Here we don't really have a choice. We have no other great way to signal "none".